### PR TITLE
fix: fix console warnings and errors for dev environment

### PR DESCRIPTION
**Pull request by ozgurkececioglu**
### Description

Fixed the parts which produce warnings and errors in the console

### How to test

- seeing green PR
- checking dev tools console on local to see if any warnings/errors exist for our code

### Development checks

- [x] Ensure that deployed demo has expected results
- [x] Ensure that tests pass by running `yarn test`
- [x] Ensure that visuals tests pass by running `yarn test:visual`. If not - check the documentation [how to fix visual tests](https://github.com/toptal/picasso/blob/master/docs/contribution/visual-testing.md#fixing-broken-visual-tests-inside-a-pr)
- [x] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run all` - Run whole pipeline
- `@toptal-bot run build` - Check build
- `@toptal-bot run visual` - Run visual tests
- `@toptal-bot run deploy:documentation` - Deploy documentation
- `@toptal-bot run package:alpha-release` - Release alpha version

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

### CODE REVIEW

File: packages/picasso/src/PageHead/story/Steps.example.tsx [hunk 1]
```diff
@@ -4,7 +4,9 @@ import { PageHead, Stepper } from '@toptal/picasso'
 const StepsExample = () => (
   <PageHead>
     <PageHead.Main>
-      <Stepper steps={['Label', 'Label', 'Label', 'Label', 'Label']} />
+      <Stepper
+        steps={['Label-1', 'Label-2', 'Label-3', 'Label-4', 'Label-5']}
```
Comment (OndrejTuma) [inline packages/picasso/src/PageHead/story/Steps.example.tsx:8]: Just need to generate visual snapshot for this one 👍

Comment (OndrejTuma) [reply packages/picasso/src/PageHead/story/Steps.example.tsx:8]: Maybe for better presentation?
```suggestion
        steps={['Label 1', 'Label 2', 'Label 3', 'Label 4', 'Label 5']}
```

File: packages/topkit-analytics-charts/src/CategoriesChart/story/Default.example.tsx [hunk 2]
```diff
@@ -223,12 +223,14 @@ const CHART_TOOLTIPS = {
...
-    width={720}
-  />
+  <div style={{ width: 720 }}>
+    <CategoriesChart
+      data={CHART_DATA}
+      labels={CHART_LABELS}
+      tooltips={CHART_TOOLTIPS}
+      width='100%'
+    />
+  </div>
```
Comment (OndrejTuma) [inline packages/topkit-analytics-charts/src/CategoriesChart/story/Default.example.tsx:233]: 👍

File: .storybook/main.js [hunk 3]
```diff
@@ -20,7 +20,6 @@ module.exports = {
     // no "/register" because https://github.com/storybookjs/storybook/issues/11929#issuecomment-672998494
     '@storybook/addon-a11y',
 
-    './addons/menu-expand/register',
```
Comment (OndrejTuma) [inline .storybook/main.js:23]: Is it safe to remove this?

Comment (ozgurkececioglu) [reply .storybook/main.js:23]: I checked and didn't see any differences. The selector inside was already broken, so I don't think there will be an issue 🙂

File: packages/picasso/src/PageHead/story/Steps.example.tsx (hunk 1 above)
Comment (ozgurkececioglu) [reply packages/picasso/src/PageHead/story/Steps.example.tsx:8]: I will add the new visual snapshot and thanks for the suggestion 👍

File: packages/picasso/src/Typography/story/Decoration.example.tsx [hunk 4]
```diff
@@ -10,7 +10,7 @@ const Example = () => (
       <Typography underline='dashed'>dashed</Typography>
     </Container>
     <Container bottom='small'>
-      <Typography>
+      <Typography as='div'>
```
Comment (TomasSlama) [inline packages/picasso/src/Typography/story/Decoration.example.tsx:13]: this looks wrong to me 🤔. I would expect that we want to have text inside paragraph and the inner typography should be inside of a span, right? So why do we need to make it div 🤔?

Comment (ozgurkececioglu) [reply packages/picasso/src/Typography/story/Decoration.example.tsx:13]: Typography uses `<p>` as the default component. And, the inner Typography also will render `<p>` and that's why we are seeing console error like `<p> cannot be inside another <p>`. I made it `div` but we can also change inner typography to `span` for example. It's just a taste of code 🙂

File: packages/picasso/src/FieldRequirements/story/Error.example.tsx [hunk 5]
```diff
@@ -25,7 +25,7 @@ const Example = () => {
   return (
     <Container>
       <Input
-        error
+        status='error'
```
Comment (denieler) [inline packages/picasso/src/FieldRequirements/story/Error.example.tsx:28]: oh, wow 😁

File: packages/picasso/src/PageHead/story/Steps.example.tsx (hunk 1 above)
Comment (denieler) [reply packages/picasso/src/PageHead/story/Steps.example.tsx:8]: Maybe use some more real-world labels? maybe we can take from real projects?

File: packages/picasso/src/Typography/story/Decoration.example.tsx (hunk 4 above)
Comment (denieler) [reply packages/picasso/src/Typography/story/Decoration.example.tsx:13]: indeed this looks wrong

File: packages/picasso/src/utils/Breakpoints/story/useScreens.example.tsx [hunk 6]
```diff
@@ -8,7 +8,7 @@ const Example = () => {
 
   return (
     <>
-      <Typography>
+      <Typography as='div'>
```
Comment (denieler) [inline packages/picasso/src/utils/Breakpoints/story/useScreens.example.tsx:11]: same here 👀

**Review by denieler — APPROVED** [review]
All good, except `Typography` as `div`

File: packages/picasso/src/OverviewBlock/story/index.jsx [hunk 7]
```diff
@@ -31,7 +31,7 @@ page
         type: 'string',
         description: 'Counter title'
       },
-      color: {
```
[the remark this answers is not in the document]
Comment (ozgurkececioglu) [reply packages/picasso/src/OverviewBlock/story/index.jsx:34]: With additional docs:
Screen Shot 2022-05-10 at 15 35 00

Without additionalDocs:
Screen Shot 2022-05-10 at 15 35 43

I don't know why 🙂

[the remark this answers is not in the document]
Comment (ozgurkececioglu) [reply packages/picasso/src/OverviewBlock/story/index.jsx:34]: You are right @denieler, additional docs are old and wrong. I will remove 👍

**Review by TomasSlama — APPROVED** [review]

**Review by dmaklygin — APPROVED** [review]

**Pull request merged**